### ISSUE TITLE
pegjs parser do not recognize delayed resync activity

### ISSUE THREAD

**Issue by fussel178** [server-state/linux-raid-module#6]
The status file `/proc/mdstat` shows a different option line comparable to the activity line, but without any progress.
```
Personalities : [raid1] 
md1 : active raid1 sdb8[1] sdb7[0]
      20954112 blocks super 1.2 [2/2] [UU]
        resync=DELAYED

unused devices: <none>

```
This can be reproduced by idling the sync action of the array:
`echo "idle" > /sys/block/md1/md/sync_action`

Currently the parser do not recognize this feature and assign to the `unknown` option.

JSON Output:
```json
{
   "personalities": [
      "raid1"
   ],
   "raids": [
      {
         "name": "md1",
         "state": "active",
         "access": "rw",
         "type": "raid1",
         "devices": [
            {
               "name": "sdb8",
               "index": 1,
               "status": "active"
            },
            {
               "name": "sdb7",
               "index": 0,
               "status": "active"
            }
         ],
         "blocks": 20954112,
         "parameters": "super 1.2",
         "ideal": 2,
         "current": 2,
         "options": [
            {
               "type": "unknown",
               "value": "resync=DELAYED"
            }
         ]
      }
   ]
}
```

**Comment by pklaschka** [comment]
Just to clarify: Should the desired JSON output be

```json
{
   "personalities": [
      "raid1"
   ],
   "raids": [
      {
         "name": "md1",
         "state": "active",
         "access": "rw",
         "type": "raid1",
         "devices": [
            {
               "name": "sdb8",
               "index": 1,
               "status": "active"
            },
            {
               "name": "sdb7",
               "index": 0,
               "status": "active"
            }
         ],
         "blocks": 20954112,
         "parameters": "super 1.2",
         "ideal": 2,
         "current": 2,
         "resync": "delayed",
         "options": []
      }
   ]
}
```

or

```json
{
   "personalities": [
      "raid1"
   ],
   "raids": [
      {
         "name": "md1",
         "state": "active",
         "access": "rw",
         "type": "raid1",
         "devices": [
            {
               "name": "sdb8",
               "index": 1,
               "status": "active"
            },
            {
               "name": "sdb7",
               "index": 0,
               "status": "active"
            }
         ],
         "blocks": 20954112,
         "parameters": "super 1.2",
         "ideal": 2,
         "current": 2,
         "options": [
            {
               "type": "resync",
               "value": "DELAYED"
            }
         ]
      }
   ]
}
```

or something completely different? Thanks in advance :+1:

**Comment by fussel178** [comment]
A bit of the last one.
I think something like this:
```json
{
   "personalities": [
      "raid1"
   ],
   "raids": [
      {
         "name": "md1",
         "state": "active",
         "access": "rw",
         "type": "raid1",
         "devices": [
            {
               "name": "sdb8",
               "index": 1,
               "status": "active"
            },
            {
               "name": "sdb7",
               "index": 0,
               "status": "active"
            }
         ],
         "blocks": 20954112,
         "parameters": "super 1.2",
         "ideal": 2,
         "current": 2,
         "options": [
            {
               "type": "activity",
               "activityType": "resync",
               "progress": "DELAYED"
            }
         ]
      }
   ]
}
```

This activity types can be delayed:
- `resync`
- `recovery`

that can be delayed.

**Comment by pklaschka** [comment]
:+1:

**Comment by fussel178** [comment]
Fixed in commit 5a32f1eadd2c1f4989e4ee04364640bf0507ab67.
